### PR TITLE
Add more consistency to operationIds

### PR DESCRIPTION
**Pull request by brettmortensen**
Adds more consistency to operationIds and the endpoint summary (which is
very similar to the operationId)

### CODE REVIEW

File: openapi/mx_platform_api_beta.yml [hunk 1]
```diff
@@ -2113,14 +2113,14 @@ paths:
...
-      summary: Request connect widget URL
+      summary: Request connect widget url
       tags:
       - mx_platform
   "/users/{user_guid}/holdings":
     get:
       description: This endpoint returns all holdings associated with the specified
         `user` across all accounts and members.
-      operationId: listHoldingsByUser
+      operationId: listHoldings
```
Comment (mcoats13) [inline openapi/mx_platform_api_beta.yml:2123]: Curious why you decided to go with this particular naming? It looks like in other instances where we are listing by a `user`, `member`, etc. you are doing `listAccountOwnersByMember`. Should we leave this as `listHoldingsByUser` to follow the same pattern?

File: openapi/mx_platform_api_beta.yml [hunk 2]
```diff
@@ -2673,7 +2673,7 @@ paths:
     post:
       description: Use this endpoint to fetch the statements associated with a particular
         member.
-      operationId: fetchStatementsByMember
+      operationId: fetchStatements
```
Comment (mcoats13) [inline openapi/mx_platform_api_beta.yml:2676]: Same here.

File: openapi/mx_platform_api_beta.yml [hunk 3]
```diff
@@ -2439,14 +2439,14 @@ paths:
...
-      summary: List account owners
+      summary: List account owners by member
       tags:
       - mx_platform
   "/users/{user_guid}/members/{member_guid}/accounts/{account_guid}":
     put:
       description: This endpoint allows you to update certain attributes of an `account`
         resource.
-      operationId: updateAccount
+      operationId: updateAccountByMember
```
Comment (mcoats13) [inline openapi/mx_platform_api_beta.yml:2449]: This also seems confusing to me. We aren't just updating an account by a `member` but by a `user` as well.

File: openapi/mx_platform_api_beta.yml (hunk 1 above)
Comment (brettmortensen) [reply openapi/mx_platform_api_beta.yml:2123]: We had a good chat about naming, but I will leave a few comments to explain our conclusion. Anything that is has a `{user_guid}` in the path is not going to include `ByUser`. The `By"resource"` will be added when there is an additional guid present in the path.

File: openapi/mx_platform_api_beta.yml (hunk 2 above)
Comment (brettmortensen) [reply openapi/mx_platform_api_beta.yml:2676]: In certain scenarios like our premium agg jobs, we decided we will have outliers that don't follow our rule above.

File: openapi/mx_platform_api_beta.yml (hunk 3 above)
Comment (brettmortensen) [reply openapi/mx_platform_api_beta.yml:2449]: Addressed in the first comment above.

**Comment by mcoats13** [comment]
:shipit:

**Pull request merged**